apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -236,7 +236,7 @@ defmodule OMG.Eth.RootChain do
   @doc """
   Returns exit for a specific utxo. Calls contract method.
   """
-  def get_exit(exit_id, contract \\ nil) do
+  def get_standard_exit(exit_id, contract \\ nil) do
```

Comment:
Regarding rename. How about [this one](https://github.com/omisego/elixir-omg/blob/fc0535bba9d09f817919969da03efc249de6c9f1/apps/omg_eth/lib/eth/root_chain.ex#L317), looks very similar, shouldn't be renamed as well?
Also, [this comment](https://github.com/omisego/elixir-omg/blob/fc0535bba9d09f817919969da03efc249de6c9f1/apps/omg_watcher/test/exit_processor/core_test.exs#L104) is missing a change.